### PR TITLE
fix: build errors

### PR DESCRIPTION
**Pull request by ConjunctiveNormalForm**
fixing some build errors in #452

### CODE REVIEW

File: lib/util/order.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const eventRecordToOrder = (record: DynamoDBRecord): ParsedOrder => {
 export const formatOrderEntity = (
   decodedOrder: DutchOrder,
   signature: string,
-  orderType: OrderType.Dutch | OrderType.Dutch_V2 | OrderType.Limit,
```
Comment (zhongeric) [inline lib/util/order.ts:51]: do we need to special case DutchV2 here then to pull the start/end times from the cosigner Data?

File: test/integ/repositories/dynamo-repository.test.ts [hunk 2]
```diff
@@ -174,7 +174,7 @@ describe('OrdersRepository put item test', () => {
   it('should succesfully put a priority order in table', async () => {
     expect(() => {
       mockTime(1)
-      ordersRepository.putOrderAndUpdateNonceTransaction(PRIORITY_ORDER_1 as UniswapXOrderEntity)
+      ordersRepository.putOrderAndUpdateNonceTransaction(PRIORITY_ORDER_1 as unknown as UniswapXOrderEntity)
```
Comment (zhongeric) [inline test/integ/repositories/dynamo-repository.test.ts:177]: I don't love the unknown cast - why doesn't PriorityOrder fit into UniswapXOrderEntity?

**Review by zhongeric — APPROVED** [review]

File: lib/util/order.ts (hunk 1 above)
Comment (ConjunctiveNormalForm) [reply lib/util/order.ts:51]: this function is only being used for v1 orders (and limit, which is also v1) so probably no need

File: test/integ/repositories/dynamo-repository.test.ts (hunk 2 above)
Comment (ConjunctiveNormalForm) [reply test/integ/repositories/dynamo-repository.test.ts:177]: discussed offline, all mock orders only contain a subset of actual order entity type fields and that's fine for testing purpose

**Pull request merged**